HackYourFuture-CPH/Guide-IT
# finished storybook job card features

# Description

Component storybook for right result job card features, with 3 bullet points. This component is called with one prop, the branch name,: Full stack development, UX design or data analysis. Depending on the result and the clicked bullet point, the user will see the needed info that is stored in an array.

Fixes #21 

# How to test?

command: npm run storybook, and click in the bullet points, you can change the control but typing a brach name and see different results
# Checklist
